Source/AI.cs

```diff
@@ -22,6 +24,9 @@ public class Input
 			public string[] PreviousHistoricalKeyEvents { get; set; }
 			public string LastSpokenText { get; set; }
 			public List<string> CurrentGameState { get; set; }
+			public string[] ColonyRoster { get; set; }
```

Comment:
I am not a native speaker so the term 'Roster' seems strange to me.

Reply:
Roster usually means a list of people, in this context I think its good.

Reply:
TIL